database/OpenRA_Red_Alert/openra-d2k.desktop

```diff
@@ -0,0 +1,20 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+Name=OpenRA - Dune 2000
```

Comment:
I noticed that you auto-generated scripts pick the first in alphabetical order. You may want to change this to 
```suggestion
Name=OpenRA - Red Alert
```
though. As it is the eponymous default mod.